### PR TITLE
Shi2018pointrcnn Summary 2

### PR DESCRIPTION
**Pull request by liaojh1998**
To be done by tonight

### CODE REVIEW

**Comment by liaojh1998** [comment]
Hey @kraja99, I just finished the first draft. Sorry for being late on this, but the summary should be up for review now. Please let me know if there are any issues!

File: _summaries/shi2018pointrcnn_2.md [hunk 1]
```diff
@@ -1,19 +1,124 @@
...
+
+<p align="center">
+    <img src="shi2018pointrcnn/refinement_loss.png" width="20%" height="20%">
+</p>
+
+Where $$\mathcal{B}$$ is the set of 3D proposals from stage-1 and $$\mathcal{B}_{pos}$$ are the positive (>0.55 IoU) proposals for regression. $$\text{prob}_{i}$$ denotes the estimated confidence of the bounding box. Eventually, another NMS is applied to remove overlapping bounding boxes with an IoU threshold of 0.01.
+
 * What interesting variants are explored?
 
+**GT-AUG**: To simulate objects with various environments, the authors copied several ground-truth boxes and their inside points from some scenes to same locations of other scenes. This slightly improved mAP on the validation set.
```
Comment (kraja99) [inline _summaries/shi2018pointrcnn_2.md:84]: Consider briefly mentioning the other data augmentations the authors used (random flip, rotation, etc.) as well as other implementation details (ex. ~16k points were subsampled from each point cloud with repeating points if the input is smaller).

File: _summaries/shi2018pointrcnn_2.md [hunk 2]
```diff
@@ -1,19 +1,124 @@
...
+</p>
+
+Full bin-based loss (BB-loss) outperformed residual-based loss and other losses in convergence:
+
+<p align="center">
+    <img src="shi2018pointrcnn/ablate3.png" width="30%" height="30%">
+</p>
+
+
 ## TL;DR
```
Comment (kraja99) [inline _summaries/shi2018pointrcnn_2.md:121]: Great summary! You do an excellent job conveying the technical aspects of the paper. Maybe add some examples of the qualitative results from Figure 6 to help visualize what the model is outputting.

**Review by kraja99 — COMMENTED** [review]
Nice Summary!

**Comment by liaojh1998** [comment]
Thanks for the summary @kraja99. I added some implementation details and figure 6 for visualizations. Let me know if there's any other issues!

**Comment by zhaoyue-zephyrus** [comment]
@liaojh1998 hi, I don't think I see any figure. Would you take a look at it?

**Comment by liaojh1998** [comment]
> @liaojh1998 hi, I don't think I see any figure. Would you take a look at it?

@zhaoyue-zephyrus I just added it! Sorry about that.

**Comment by zhaoyue-zephyrus** [comment]
LGTM. Thank you @liaojh1998 .

**Review by zhaoyue-zephyrus — APPROVED** [review]
